src/OpenDebugAD7/AD7DebugSession.cs

```diff
@@ -682,7 +683,7 @@ protected override void HandleInitializeRequestAsync(IRequestResponder<Initializ
             }
 
             List<ColumnDescriptor> additionalModuleColumns = null;
-            string clientId = responder.Arguments.ClientID;
+            clientId = responder.Arguments.ClientID;
```

Comment:
>clientId [](start = 12, length = 8)

Rather than storing 'clientId' you might want to instead have a 'm_isClientVS` which would be set to true in either of these cases. We will might someday have a case where we want to differentiate "visualstudio" and "liveshare-server-host", but so far all of the cases would want to treat both the same (including your new check).

Reply:
I think its about time we have m_clientId enum.
```
 enum ClientID
{
   VisualStudio,
   VsCode,
   LiveShareServerHost,
};
```

Then we can have a property `IsClientVS` as
```
bool IsClientVS {
   get {
      return m_clientId == ClientID.VisualStudio || m_clientId == ClientID.LiveShareServerHost;
   }
}
```

Reply:
Sounds fine to me

Reply:
None that you need an 'Unknown' value to for ClientID.